docs/1.2/core/reference/cored.md

```diff
@@ -83,6 +83,13 @@ the limit will receive an HTTP 429 response.
 
     Can be stacked with **RATELIMIT_TOKEN**.
 
+* **BOOTURL**: Setting this value causes the `cored` process to join an
```

Comment:
We [decided not to keep the environment variable](https://github.com/chain/chain/pull/1229#issuecomment-303261089) for backwards-compatibility because it's only used when setting up a new cluster.